src/utils/getRelevantMultisig.ts

```diff
@@ -0,0 +1,19 @@
+import type { AccountInfo, MultisigInfo } from '../types/account';
+import type { Bounty } from '../types/bounty';
+import type { ChildBounty } from '../types/child-bounty';
+
+export function getRelevantMultisig(
+	bounty: Bounty | ChildBounty,
+	account: AccountInfo | undefined
+): MultisigInfo | undefined {
+	if (!account) {
+		return undefined;
+	}
```

Comment:
```suggestion
	account: AccountInfo
): MultisigInfo | undefined {
```
This is used in one place only and the account is defined there